torchao/experimental/ops/packed_weights_header.h

```diff
@@ -10,44 +10,47 @@
 #include <cassert>
 namespace torchao::ops {
 
-enum PackedWeightsFormat : unsigned short {
+enum PackedWeightsFormat : int {
   unknown = 0,
   linear_8bit_act_xbit_weight_universal = 1
 };
 
 class PackedWeightsHeader {
  public:
-  using params_type = std::array<unsigned short, 7>;
+  using params_type = std::array<int, 14>;
+  const static int magic = 6712;
   PackedWeightsFormat format;
 
   // 14 bytes of format specific params
   params_type params;
 
   PackedWeightsHeader(
       PackedWeightsFormat format = PackedWeightsFormat::unknown,
-      params_type params = {0, 0, 0, 0, 0, 0, 0})
+      params_type params = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0})
       : format{format}, params{params} {}
 
   inline static constexpr int size() {
-    static_assert(sizeof(format) + sizeof(params) == 16);
-    return 16;
+    static_assert(sizeof(magic) + sizeof(format) + sizeof(params) == 64);
+    return 64;
```

Comment:
Why not use sizeof(XYZ)?
```suggestion
    return sizeof(PackedWeightsHeader);
```

Reply:
I thought this made it clearer that the header is 64 bytes at a glance.